src/client/pages/RetroCertsConfirmationPage/index.js

```diff
@@ -1,7 +1,7 @@
 import Button from "react-bootstrap/Button";
```

Comment:
I think we need to a print CSS style to show all the weeks.

Reply:
Talked to Tina, we'll implement expanding them all when printing as an enhancement in the next PR.